0.1-test-notebook.ipynb

```diff
@@ -0,0 +1,200 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Welcome to Jupyter\n",
+    "\n",
+    "This is a Jupyer Notebook. It contains both computer code (e.g. python) and rich text elements (paragraph, equations, figures, links, etc…). This paragraph is an example of the rich test element. You can double click on this paragraph 'cell' to edit it. \n",
+    "\n",
+    "Below is a code block. You can double click to edit it. You can also run the code step-by-step (one cell a time) by pressing shift + enter when you have the cell selected."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "21"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "14+7"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Configure your source directory\n",
+    "\n",
+    "In the `setup.py` file, enter the source directory for your local Optech taproot (v0.1 tag) bitcoin repo.  \n",
+    "This will be used to access the TestFramework Python code and the taproot-compatible bitcoind binary."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Source directory configured as /Users/schmidty/Dropbox/Repos/bitcoin-optech-taproot\n"
+     ]
+    }
+   ],
+   "source": [
+    "import setup"
```

Comment:
Update to `import util` when [TestWrapper PR](https://github.com/bitcoinops/taproot-workshop/pull/8) is resolved.

Reply:
Done. I've rebased this PR on the TestWrapper changes.